tests/Microsoft.Bot.Builder.LanguageGeneration.Tests/TemplatesTest.cs

```diff
@@ -1093,6 +1093,21 @@ public void TestNullTolerant()
             Assert.AreEqual("null", jObjEvaled["key1"]);
         }
 
+        [TestMethod]
+        public void TestLGOptions()
+        {
+            var eof = Environment.NewLine;
```

Comment:
do not use `Environment.NewLine` here, because u do not know the newline tyle of lg content